testsuite/classlibrary/TestNodeProxy.sc

```diff
@@ -40,4 +40,103 @@ TestNodeProxy : UnitTest {
 		this.assertEquals(proxy.isPlaying, false, "Setting the proxy's source should not set isPlaying = true");
 	}
 
+	test_asCode_basic {
+		var codeString = "a = NodeProxy.new(Server.fromName( 'TestNodeProxy' ));\n";
+
+		this.assertEquals(proxy.asCode.postcs, codeString,
+			"asCode-posting basic nodeproxy should post valid source code.",
+			false);
+	}
+
+	test_asCode_single {
+		var codeString = "a = NodeProxy.new(Server.fromName( 'TestNodeProxy' )).source_({ DC.ar });\n";
+		proxy.source = { DC.ar };
+
+		this.assertEquals(proxy.asCode, codeString,
+			"asCode-posting single-source nodeproxy should post simple source form.",
+			false
```

Comment:
why are some of these set to `false` and others not?

Reply:
I was experimenting with suppressing reports if true, and only found that UnitTest.passVerbosity allows that afterwards. So, all falses removed now.